### PR TITLE
Publication image block

### LINKED ISSUE #6275: Create an image block for Article pages
Editors need a way to show images with a caption. Instead of overwriting wagtail core features, let's create a new image block for full width images. This shouldn't be confused with the ultra-wide image that's currently available that reaches both edges of the screen. This new image block should be the equivalent of selecting a "full width" image in the rich text editor, with the option to add a caption

Acceptance criteria:
- [ ] Need new "full width" image block in Article Pages. 
- [ ] Needs an image chooser
- [ ] Needs to have a richtext caption (with bold, italics, links)
- [ ] Needs an alt tag charfield that defaults to the image title if it's not provided.

### PR DESCRIPTION
**Pull request by KalobTaulien**
Closes #6275 

This PR introduces a standard "image" block for Article pages. It holds an image, optional caption and optional alt text. 

It's _very_ similar to the ImageBlock we already have, but has an alt text field.

**Link to sample test page**: https://foundation-s-6275-publi-9vdstc.herokuapp.com/en/publication-page-with-child-article-pages/fixed-title-article-page/

Acceptance criteria:
- [x] Need new ~"full width"~ image block in Article Pages. (I named it "image block")
- [x] Needs an image chooser
- [x] Needs to have a richtext caption (with bold, italics, links)
- [x] Needs an alt tag charfield that defaults to the image title if it's not provided.

Preview:
image

### CODE REVIEW

**Review by sabrinang — COMMENTED** [review]
Image block and caption looks great and works. 

For the previous full width image option, when users wanted to upload images with text there was the "Full width high quality" to solve the pixelation issue. Would that be an option for this image block?

**Comment by KalobTaulien** [comment]
Hey @sabrinang no that's not an option here. We can automatically add that in by doubling the pixels or using srcsets. I'll make sure the image is 2x the resolution in the next commit.

**Review by sabrinang — APPROVED** [review]
Text looks sharp now on the image block — awesome, thank you Kalob!

**Review by mmmavis — CHANGES_REQUESTED** [review]
Code looks good 🎉 but I'm gonna test it one more time after migration file conflict has been fixed.

**Review by recepsirin — APPROVED** [review]

**Comment by KalobTaulien** [comment]
@mmmavis Fixed the migration :tada:

File: network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/article_image_block.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+{% comment %}This block doesn't need to "break" out of the Publications container{% endcomment %}
+<div class="article-image-block">
+  {% comment %}Use a 2x resolution image{% endcomment %}
+  {% image self.image width-1840 as img %}
+  <figure class="my-default">
+    <img src="{{ img.url }}" alt="{% if self.alt_text %}{{ self.alt_text }}{% else %}{{ img.alt }}{% endif %}" class="w-100" />
+    {% if self.caption %}
+        <figurecaption>
+          {{ self.caption|richtext|remove_wrapping|safe }}
+        </figurecaption>
```
Comment (mmmavis) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/article_image_block.html:12]:
```suggestion
        <figcaption>
          {{ self.caption|richtext|remove_wrapping|safe }}
        </figcaption>
```

File: source/sass/components/article-page.scss [hunk 2]
```diff
@@ -332,6 +332,7 @@
   }
 }
 
+.article-image-block,
 .double-image-block,
 .full-width-image-block {
   figurecaption p {
```
Comment (mmmavis) [inline source/sass/components/article-page.scss:338]: might wanna add `figcaption p` here as a selector first

I'll file a new ticket to replace all `figurecaption` instances with `figcaption` 😂

**Review by mmmavis — CHANGES_REQUESTED** [review]
haha can't believe we've been all using `figurecpation` for `figcaption`

File: network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/article_image_block.html (hunk 1 above)
Comment (KalobTaulien) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/article_image_block.html:12]: LOL that's funny this was never caught until now. Good catch Mavis!

**Review by mmmavis — APPROVED** [review]
Code changes look good 🎉 . A few tests failed so will need to address those issues before this can be merged.

**Comment by KalobTaulien** [comment]
Migration conflicts. I'll bump the migration, run tests, and push the fix shortly.

**Pull request merged**